### PR TITLE
Adapt Instructor notes

### PR DESCRIPTION
**Pull request by sdruskat**
This PR adapts the instructor notes document to work with the offline setting.

There are still a couple of pitfalls:

- Unclear what we want to suggest to instructors to do with online resources (websites, Raniere's shell split script, etc.). There's a callout noting that resources should ideally be downloaded beforehand. But is this the best option?
- Link to offline Gitea on Pi is assumed to be `http://carpentriesoffline.org:3000`.

### CODE REVIEW

File: instructors/instructor-notes.md [hunk 1]
```diff
@@ -181,8 +191,8 @@ particular set of files in `.gitignore`.
 - When pushing to a remote, the output from Git can vary slightly depending on
   what leaners execute. The lesson displays the output from git if a learner
   executes `git push origin main`. However, some learners might use syntax
-  suggested by GitHub for pushing to a remote with an existing repository,
-  which is `git push -u origin main`. Learners using syntax from GitHub,
+  suggested by Gitea for pushing to a remote with an existing repository,
+  which is `git push -u origin main`. Learners using syntax from Gitea,
   `git push -u origin main`, will have slightly different output, including
```
Comment (pete-arnold) [inline instructors/instructor-notes.md]:
```suggestion
  which is `git push -u origin main`. Learners using this syntax from Gitea,
  will have slightly different output, including
```
The duplication of the command unduly worried me as I wasn't sure if they were supposed to be different.

File: instructors/instructor-notes.md [hunk 2]
```diff
@@ -260,7 +270,7 @@ particular set of files in `.gitignore`.
 - If you're the only instructor, the best way to create a conflict is:
   
   - Clone your repo in a different directory, pretending is your computer at
```
Comment (pete-arnold) [inline instructors/instructor-notes.md]:
```suggestion
  - Clone your repo in a different directory, pretending it is your computer at
```

**Review by pete-arnold — APPROVED** [review]
Reviewed all the changes - see comments which include fixes.

**Comment by sdruskat** [comment]
Thanks @pete-arnold for the review and fixes, co-committed all suggestions.
Happy for others (@colinsauze, @jsteyn?) to review as well before merging.

**Comment by abhidg** [comment]
Thanks @sdruskat, I have merged this!

**Pull request merged**